docs/core/tutorial/06-parallel-execution.md

```diff
@@ -0,0 +1,34 @@
+# Parallel Execution
+
+::: tip Follow along in the Terminal
+
+```
+cd examples/tutorial
+python 06_parallel_execution.py
+```
+
+:::
+
+Let's adjust our Flow to distribute our Tasks onto a [Dask](https://dask.org/) Cluster, parallizing our execution. This may sound involved, but will actually be our simplest adjustment yet:
```

Comment:
```suggestion
Let's adjust our Flow to distribute our Tasks onto a [Dask](https://dask.org/) Cluster, parallelizing our execution. This may sound involved, but will actually be our simplest adjustment yet:
```